### PR TITLE
[cxx-interop] Document new macros from `<swift/bridging>` header

### PR DESCRIPTION
**Pull request by egorzhdan**
This adds `SWIFT_MUTATING`, `SWIFT_NONCOPYABLE` to the list of supported customization macros.

### CODE REVIEW

**Comment by egorzhdan** [comment]
cc @Xazax-hun @ravikandhadai (I can't seem to add you as reviewers)

File: documentation/cxx-interop/index.md [hunk 1]
```diff
@@ -418,12 +421,9 @@ of better API usability Swift still assumes that such functions do not
...
 that mutate `mutable` fields from Swift, unless they're explicitly annotated with a `SWIFT_MUTATING` macro.
 
-> Swift 5.9 will ship with a `SWIFT_MUTATING` customization macro. However, it's not
-> yet available in a downloadable Swift 5.9 toolchain. This macro will
-> allow you to explicitly annotate constant member functions that do mutate the
-> object. Such functions with then become `mutating` methods in Swift. The
-> following [GitHub issue](https://github.com/swiftlang/swift/issues/66322)
-> tracks the status of `SWIFT_MUTATING` support in Swift.
+The `SWIFT_MUTATING` macro allows you to explicitly annotate constant member
+functions that do mutate the object. Such functions with then become `mutating`
```
Comment (Xazax-hun) [inline documentation/cxx-interop/index.md:425]: with -> will? But maybe even better to use simple present like "Swift imports such functions as `mutating` methods".

Comment (egorzhdan) [reply documentation/cxx-interop/index.md]: Oh right, thanks for noticing! I removed the word `with`.

**Comment by egorzhdan** [comment]
@swift-ci please test

**Comment by egorzhdan** [comment]
@swift-ci please test

**Review by parispittman — APPROVED** [review]

**Pull request merged**